accounts/ethstore/src/ethstore.rs

```diff
@@ -37,11 +38,11 @@ pub struct EthStore {
 impl EthStore {
 	/// Open a new accounts store with given key directory backend.
 	pub fn open(directory: Box<KeyDirectory>) -> Result<Self, Error> {
-		Self::open_with_iterations(directory, KEY_ITERATIONS as u32)
+		Self::open_with_iterations(directory, NonZeroU32::new(KEY_ITERATIONS as u32).expect("KEY_ITERATIONS > 0; qed"))
```

Comment:
This is just a `constant` and could be `unchecked` too? 
What's the reasoning on determining whether `unchecked` or not?

I guess it makes sense in this case because `KEY_ITERATIONS` is imported from `parity-crypto` right?

Reply:
Yes, you're right. I initially used `new_unchecked` only for literals, where is was clear it's safe. But since `crypto::KEY_ITERATIONS` is a constant, it's safe too.